### PR TITLE
update external test to run with docker images provided by adoptium

### PR DESCRIPTION
**Pull request by LizyBbethy**
Added cleanup and update of files to latest  official adoptium images of the Eclipse Temurin Distribution

issue 3116
Signed-off by; LizyBbethy @LizyBbethy

### CODE REVIEW

**Review by jnjerin — APPROVED** [review]
Just reviewed this, @LizyBbethy has done a good job 👏

File: external/derby/test.properties [hunk 1]
```diff
@@ -3,7 +3,4 @@ script="derby-test.sh"
 home_path="\${WORKDIR}/derby"
 tag_version="trunk"
 ant_version="1.10.6"
-debian_packages="git wget tar"
-alpine_packages="bash git wget tar"
-centos_packages="git wget tar"
-ubi_packages="git wget tar"
+ubuntu_packages="git maven"
```
Comment (sophia-guo) [inline external/derby/test.properties:6]: ubuntu_packages should be same as debian_packages. In this case `ubuntu_packages="git wget tar"`. Same for others.

**Comment by sophia-guo** [comment]
ubuntu_packages should be same as debian_packages. It may not be `git maven` for some third party applications, which depends on the third party application's requirements or dependencies.

**Comment by jnjerin** [comment]
> ubuntu_packages should be same as debian_packages. It may not be `git maven` for some third party applications, which depends on the third party application's requirements or dependencies.

@sophia-guo Thanks for the feedback. How do we confirm the specific 3rd party application's dependencies required to be included?

File: external/derby/test.properties (hunk 1 above)
Comment (LizyBbethy) [reply external/derby/test.properties:6]: thank you @sophia-guo for the review. does that neccesitate following the debian_packages logic as used

**Comment by LizyBbethy** [comment]
> ubuntu_packages should be same as debian_packages. It may not be `git maven` for some third party applications, which depends on the third party application's requirements or dependencies.

As i look into changes at bottom of some files to allow for github collaborations

**Comment by sophia-guo** [comment]
Basically for each test.properties files replacing the key debian_packages with ubuntu_packages should be enough. The change you did in the common_function.sh is correct.

**Comment by LizyBbethy** [comment]
> Basically for each test.properties files replacing the key debian_packages with ubuntu_packages should be enough. The change you did in the common_function.sh is correct.

hi @sophia-guo . Do u have any more changes that  i should  consider  to push in ?i am working on the issue

**Review by smlambert — COMMENTED** [review]
This looks good, we should run some test jobs against it.  I am having 2nd thoughts on my request for the temporary removal of openj9, as I understand now that they are actually running these tests.  I will request that we keep line 20 in common_functions.hs unchanged, so as not to disrupt the openj9 testing.

File: external/common_functions.sh [hunk 2]
```diff
@@ -14,20 +14,20 @@
 #
 
 # Supported versions
-supported_versions="8 9 10 11 12 13 14 15 16 17"
+supported_versions="8 11 17"
 
 # Supported JVMs
-supported_jvms="hotspot openj9"
+supported_jvms="hotspot"
```
Comment (smlambert) [inline external/common_functions.sh:20]: Changing the plan on the original request, let's keep openj9 in this line.

**Comment by LizyBbethy** [comment]
Hi @smlambert @sophia-guo @llxia

**Comment by smlambert** [comment]
jacoco Grinder at: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2537

**Comment by LizyBbethy** [comment]
jacoco Grinder at ; https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2539/   with jdk 11

**Review by smlambert — COMMENTED** [review]
The changes to the files in the external folder look good, but I do not think you should have any changes in the system/security folder.

**Comment by LizyBbethy** [comment]
Hi @smlambert  am still failing to see the changes you’ve said in system/ security folder for correction

**Comment by smlambert** [comment]
hi @LizyBbethy - I am looking at: https://github.com/adoptium/aqa-tests/pull/3131/files, scrolling down, of the 25 files you have changed in your PR, 2 files at the bottom of the list were merged into master already by a different contributor as part of https://github.com/adoptium/aqa-tests/pull/2970, so don't think they should be part of your PR, you may want to update your branch to be in sync with latest from master.  

Screen Shot 2021-11-17 at 8 15 59 AM

**Comment by LizyBbethy** [comment]
> hi @LizyBbethy - I am looking at: https://github.com/adoptium/aqa-tests/pull/3131/files, scrolling down, of the 25 files you have changed in your PR, 2 files at the bottom of the list were merged into master already by a different contributor as part of #2970, so don't think they should be part of your PR, you may want to update your branch to be in sync with latest from master.
> 
> Screen Shot 2021-11-17 at 8 15 59 AM

thank you . working my way around it .

**Comment by LizyBbethy** [comment]
corrections made in PR https://github.com/adoptium/aqa-tests/pull/3143/files

**Pull request closed without merging**